engine/src/flutter/impeller/renderer/backend/vulkan/swapchain/ahb/ahb_swapchain_impl_vk.cc

```diff
@@ -67,6 +89,11 @@ AHBSwapchainImplVK::AHBSwapchainImplVK(
 
   auto control = surface_control_.lock();
   is_valid_ = control && control->IsValid();
+
+  for (auto i = 0u; i < kMaxPendingPresents; i++) {
+    frame_data_.push_back(std::make_unique<AHBFrameSynchronizerVK>(
```

Comment:
Let's add some error checking here? Perhaps a validity checks for all the handles.

Reply:
Done